streamer/src/quic.rs

```diff
@@ -411,12 +411,44 @@ pub fn spawn_server(
     wait_for_chunk_timeout: Duration,
     coalesce: Duration,
 ) -> Result<(Endpoint, thread::JoinHandle<()>), QuicServerError> {
+    spawn_server_multi(
```

Comment:
The return type of this function should include `Vec<Endpoint>` right @t-nelson ?

Reply:
i wanna say, no?

@alessandrod do you have opinions?

Reply:
I mentioned it because the v1.18 BP returns a `Vec` in `SpawnServerResult`:
https://github.com/anza-xyz/agave/pull/641/files#diff-57395e2b5571717afe58b9d915860a0a7cccc56a30e5fc3099f10b85ded3aa4eR40

`SpawnServerResult` is returned by both `spawn_server()` and `spawn_server_multi()`. I'm not as familiar with the changes so I'll certainly defer to y'all; just noticed this slight difference between v1.17 and v1.18 while trying to get CI running earlier

Reply:
It doesn't really matter. The returned endpoints are used by some tests which shouldn't be using them anyway, but should be creating new endpoints. If making it return a vec requires updating 100 tests, it's not worth the trouble.